app/models/camp.rb

```diff
@@ -145,10 +145,8 @@ class Camp < ActiveRecord::Base
   }
 
   scope :displayed, -> {
-    q = default_select.joins("LEFT JOIN roles ON (roles.identifier = '#{:manager}')")
-            .joins("LEFT JOIN people ON (people.camp_id = camps.id)")
-            .joins("LEFT JOIN people_roles pr ON (pr.role_id = roles.id)")
-            .where('people.id = pr.person_id')
+    q = default_select.joins("LEFT JOIN people")
```

Comment:
Better to use `LEFT JOIN people ON (people.id = ...)`

Reply:
@sh0oki Just for my knowledge, why is it better? I have looked around for the difference and `ON` might return empty rows that `WHERE` won't.

Reply:
after adding 
```
    q = default_select.joins("LEFT JOIN people ON (people.id = camps.camp_manager_id)")
```

It currently does't work for me and just returns all the camps, if a camp's `camp_manager_id` is `nil`  it is returned, @sh0oki Can you see what I am doing wrong?
I have added tests to highlight this